packages/okta-react/test/jest/authService.test.js

```diff
@@ -252,61 +252,46 @@ describe('AuthService', () => {
 
   describe('logout', () => {
 
-    test('by default, it will set history location to "/"', async () => {
+    test('defaults to passing an empty options to signOut', async () => {
       const authService = new AuthService({
         issuer: 'https://foo/oauth2/default',
         client_id: 'foo',
```

Comment:
FYI: we are no longer accepting these "deprecated" config names with underscores in AngularJS (client_id should be clientId). We should be phasing these out across the board, and remove support in the next major version of React (is this that version?)

Reply:
Yes, this is the 3.0 release for React.  I'll update the tests and check to see if we're explicitly converting in the code.

Reply:
Config is "normalized" (with legacy support) here: https://github.com/okta/okta-oidc-js/pull/715/files#diff-8a51cbb87b5cfd7e092f0410797f00e9R30 We can skip this call if we are ready to deprecate the legacy format.